File: String.Extension.vb
Imports System.Runtime.CompilerServices

Module StringExtension

    ''' <summary>
    ''' String Join
    ''' </summary>
    ''' <typeparam name="T"></typeparam>
    ''' <param name="values"></param>
    ''' <param name="separator"></param>
    ''' <returns></returns>
    <Extension>
    Function Join(Of T)(values As IEnumerable(Of T), separator As String)
        Return String.Join(separator, values)
    End Function
End Module

File: SqlNanite/Extensions/StringExtension.cs
using System;
using System.Collections.Generic;

/// <summary>
/// The string extension.
/// </summary>
public static class StringExtension
{
    /// <summary>
    /// The join.
    /// </summary>
    /// <param name="values">
    /// The values.
    /// </param>
    /// <param name="separator">
    /// The separator.
    /// </param>
    /// <typeparam name="T">
    /// The target <see cref="Type"/>.
    /// </typeparam>
    /// <returns>
    /// The <see cref="string"/>.
    /// </returns>
    public static string Join<T>(this IEnumerable<T> values, string separator)
    {
        return string.Join(separator, values);
    }
}